pkg/apis/flowcontrol/v1beta3/defaults.go

```diff
@@ -40,7 +40,24 @@ func SetDefaults_FlowSchemaSpec(spec *v1beta3.FlowSchemaSpec) {
 	}
 }
 
-func SetDefaults_ExemptPriorityLevelConfiguration(eplc *v1beta3.ExemptPriorityLevelConfiguration) {
```

Comment:
you can keep this function as-is, SetObjectDefaults_PriorityLevelConfiguration will call both SetDefaults_PriorityLevelConfiguration and SetDefaults_ExemptPriorityLevelConfiguration